### ISSUE TITLE
Make checks less strict

### ISSUE THREAD

**Issue by IgnacioHeredia** [deephdc/deep-check_oc_artifact#1]
We should only check for DEEPaaS detecting the module. We should **NOT** check for `get_metadata()`, because:
* users might not have defined `get_metadata()`, and this is _fine_ (it should not be a hard requirement)
* users might have defined `get_metadata()` with different keys than the ones we are [checking](https://github.com/deephdc/deep-check_oc_artifact/blob/d9784e00e4fd01a89c2407e3f17d013518f8c704/check_metadata.sh#L21), and they are free to do so (it should not be a hard requirement either to use our keys)
  
Once this is done we can [reenable the checks](https://github.com/deephdc/cookiecutter-deep/commit/add7d660460f66da5e7785eff3f38166bd34d6ff) in the `master` branch of the [new Cookicutter template](https://github.com/deephdc/cookiecutter-deep).
Checks are still present in the `advanced` branch (as `get_metadata()` is indeed predefined there).

**Comment by vykozlov** [comment]
Yes and no :-)
Freedom is good but the standards do exist. I think, get_metadata() has to be **always** defined, and we should define obligatory fields in that metadata. However, which ones is an open question, but for example, license could be one of them

**Comment by IgnacioHeredia** [comment]
I agree that every user should define metadata. It's good practice. But deepaas **can** run without defining `get_metadata()` (I just tested). You can have only `predict` or even no functions at all, the module is still detected and loaded. So I guess at the time being it is not a hard requirement (at least from a functional POV). Though we could make it so, if we agree on that.

For the time being I have disabled it from the cookiecutter as this will likely add yet another source of errors for new users. And I am already spending way to much time on giving support to them.

Once the project becomes active again in September we can discuss this issue again.

**Comment by vykozlov** [comment]
You know, I am a nice guy, this is why I disagree :-)  The metadata is important and good practice. I suggest to have a TelCo among the involved people and talk about minimal template
